contrib/local-environment/oauth2-proxy-traefik.cfg

```diff
@@ -0,0 +1,23 @@
+http_address="0.0.0.0:4180"
+cookie_secret="OQINaROshtE9TcZkNAm-5Zs2Pv3xaWytBmc5W7sPX7w="
+provider="oidc"
+email_domains=["example.com"]
+oidc_issuer_url="http://dex.localhost:4190/dex"
+client_secret="b2F1dGgyLXByb3h5LWNsaWVudC1zZWNyZXQK"
+client_id="oauth2-proxy"
+cookie_secure="false"
+
+# keycloak provider
```

Comment:
Is this still true? Can we double check what's in the normal config and make this look as similar as possible?